src/views/Login/Login.scss

```diff
@@ -79,7 +79,7 @@
   .auth {
     display: inline-block;
     vertical-align: top;
-    width: 48%;
```

Comment:
set a max-width instead of width because at small screens things got a little too squished